sdk/keyvault/azure-security-keyvault-certificates/migration_guide.md

```diff
@@ -0,0 +1,260 @@
+# Guide for migrating to azure-security-keyvault-certificates from azure-keyvault
+This guide is intended to assist in the migration to `azure-security-keyvault-certificates` from `azure-keyvault`. It will focus on side-by-side comparisons for similar operations between the two packages.
+
+Familiarity with the `azure-keyvault` package is assumed. For those new to the Key Vault Certificate client library for Java, please refer to the [README](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/keyvault/azure-security-keyvault-certificates/README.md) rather than this guide.
+
+## Table of contents
+- [Migration benefits](#migration-benefits)
+- [Cross Service SDK improvements](#cross-service-sdk-improvements)
+- [Important changes](#important-changes)
+    - [Separate packages and clients](#separate-packages-and-clients)
+    - [Package names and namespaces](#package-names-and-namespaces)
+    - [Client instantiation](#client-instantiation)
+    - [Authentication](#authentication)
+    - [Common scenarios](#common-scenarios)
+        - [Async operations](#async-operations)
+        - [Create a certificate](#create-a-certificate)
+        - [Import a certificate](#import-a-certificate)
+        - [Retrieve a certificate](#retrieve-a-certificate)
+        - [List properties of certificates](#list-properties-of-certificates)
+        - [Delete a certificate](#delete-a-certificate)
+- [Additional samples](#additional-samples)
+
+## Migration benefits
```

Comment:
Usually it is better to let one space between the tittle and the paragraph.
```suggestion
## Migration benefits

```

Consider this as a nit for all the tittles on this doc

Reply:
Markdown linter rule MD022:
https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md022---headings-should-be-surrounded-by-blank-lines

Reply:
Thanks!